mongodb_migrations/cli.py

```diff
@@ -41,32 +41,59 @@ def run(self):
             print("No previous migrations found")
 
         sys.path.insert(0, self.config.mongo_migrations_path)
-        self._domigrate()
```

Comment:
we should put an underscore between 2 words, so here it should be `_do_migrate()`

Reply:
Yes, that's better!